core/test/libraries/Volatility.t.sol

```diff
@@ -177,14 +177,14 @@ contract VolatilityTest is Test {
 
     function test_spec_computeTickTVL() public {
         uint256 tickTVL;
-        tickTVL = Volatility.computeTickTVLX64(1, 19000, TickMath.getSqrtRatioAtTick(19000), 100000000000);
-        assertEq(tickTVL, 238460396558056720196173824);
-        tickTVL = Volatility.computeTickTVLX64(10, 19000, TickMath.getSqrtRatioAtTick(19000), 9763248618769789);
-        assertEq(tickTVL, 232762454487181009148555451957248);
-        tickTVL = Volatility.computeTickTVLX64(60, -19000, TickMath.getSqrtRatioAtTick(-19000), 100000000000);
-        assertEq(tickTVL, 2138446074761944812648136704);
-        tickTVL = Volatility.computeTickTVLX64(60, -3000, TickMath.getSqrtRatioAtTick(-3000), 999999999);
-        assertEq(tickTVL, 47558380999913911951032320);
+        tickTVL = Volatility.computeTickTvl(1, 19000, TickMath.getSqrtRatioAtTick(19000), 100000000000);
+        assertEq(tickTVL, 12926964);
+        tickTVL = Volatility.computeTickTvl(10, 19000, TickMath.getSqrtRatioAtTick(19000), 9763248618769789);
+        assertEq(tickTVL, 12618077941403);
+        tickTVL = Volatility.computeTickTvl(60, -19000, TickMath.getSqrtRatioAtTick(-19000), 100000000000);
+        assertEq(tickTVL, 115925394);
+        tickTVL = Volatility.computeTickTvl(60, -3000, TickMath.getSqrtRatioAtTick(-3000), 999999999);
+        assertEq(tickTVL, 2578145);
```

Comment:
New values are just the old ones, divided by 2^64. Not actually losing precision here, just not adding fake precision via bit shifts